Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -2229,13 +2232,16 @@ - (void)collectionView:(JSQMessagesCollectionView *)collectionView
     [self.collectionView layoutSubviews];
 
     self.collectionView.contentOffset = CGPointMake(0, self.collectionView.contentSize.height - scrollDistanceToBottom);
+
     [self.scrollLaterTimer invalidate];
-    // We want to scroll to the bottom _after_ the layout has been updated.
-    self.scrollLaterTimer = [NSTimer weakScheduledTimerWithTimeInterval:0.001f
-                                                                 target:self
-                                                               selector:@selector(scrollToUnreadIndicatorAnimated)
-                                                               userInfo:nil
-                                                                repeats:NO];
+    if (hasMoreUnseenMessages) {
```

Comment:
I had to read the PR comment a second time to understand what this was doing, but it totally makes sense. 

Take it or leave it, but since this re:'loadEarlierMessages' maybe calling this `hasEarlierUnseenMessages` would be clearer.